app/translations/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import * as React from 'react';
+import { Text } from '@kiwicom/react-native-app-shared';
+
+import Translation from './src/Translation';
+
+export default Translation;
+
+export type TranslationType =
+  | React.Element<typeof Translation>
+  | React.Element<typeof DummyTranslation>
+  | React.Element<typeof TranslationFragment>;
+
+export class TranslationFragment extends React.Component<{|
```

Comment:
Can we put this in its own file? If I do a file search for TranslationFragment, I wont find it.